miscellaneous-commands/subcommand.md

```diff
@@ -0,0 +1,38 @@
+# Sub Commands
+
+These commands are too minor to have thier own pages, so they are listed here.
+
+## Help
+
+```text
+/help <command>
+```
+
+* `command`: Shows information about a particular command.
+
+This command shows a list of commands, or info about a specific command.
+
+MoonlightBot requires the following permissions to successfully execute this subcommand:
+_No specific permissions required_
+
+### Ping
```

Comment:
```suggestion
## ping
```

All command headers need to be h2, incrementing the n° of #s everytime makes it look like subsections of the previous command